lib/mongo/bulk_write.ex

```diff
@@ -399,7 +399,7 @@ defmodule Mongo.BulkWrite do
 
     [q: filter,
      limit: Keyword.get(opts, :limit),
-     collation: Keyword.get(opts, :collaction)] |> filter_nils()
+     collation: Keyword.get(opts, :collection)] |> filter_nils()
```

Comment:
I guess, this was accidentally changed. The option key is `collation`.

Reply:
Fixed.

Reply:
Fixed.